packages/default-theme/nuxt.config.js

```diff
@@ -30,7 +30,7 @@ export default {
   /*
    ** Plugins to load before mounting the App
    */
-  plugins: ['~/plugins/api-client'],
+  plugins: ['~/plugins/api-client', '~/plugins/price-filter'],
```

Comment:
this has no effect, we can remove it from here
```suggestion
  plugins: [],
```

Reply:
:+1: